docs/circle.yml

```diff
@@ -0,0 +1,26 @@
+machine:
```

Comment:
Currently we use Travis for CI, do we need to introduce circle as well?

Reply:
No - this file is unnecessary